src/coreclr/vm/perfmap.cpp

```diff
@@ -121,7 +137,7 @@ PerfMap::PerfMap()
     // Initialize with no failures.
     m_ErrorEncountered = false;
 
-    m_StubsMapped = 0;
+    s_StubsMapped = 0;
```

Comment:
This is OK, but misleading. The static value was already initialized to zero and then we are conditionally re-initializing it. I'd leave it alone in the servicing change, but in the product code going forward there is no need for us to have this spurious initialization.